schema/spacewalk/common/tables/suseCredentials.sql

```diff
@@ -26,7 +26,7 @@ CREATE TABLE suseCredentials
                  REFERENCES suseCredentialsType (id),
     url      VARCHAR(256),
     username VARCHAR(64) NOT NULL,
-    password VARCHAR(64) NOT NULL,
+    password VARCHAR(1031) NOT NULL,
```

Comment:
Do not calculate the exact number of bytes. Next time it will be one more and we have the same problem again.
If it started to become normal to use passwords with more than 64 characters, just go for 2048 or 4096 directly.
It is varchar and should not need more space.

Reply:
You are right. Fixed